src/GuiHandler.java

```diff
@@ -0,0 +1,373 @@
+    	if(txtTryALetter.getText().matches(".*\\d.*")) {errorPopupHandler(0, "", "Het woord mag alleen letters bevatten"); return;}
+    	if(answer.length() <= 0) {errorPopupHandler(0, "", "Voer een woord in om te raden"); return;}
+    	
+        String letter = txtTryALetter.getText();
+        int inputLenght = letter.length();
+        
+        try 
+        {
+        	if(inputLenght > 1 || inputLenght <= 0) throw new IllegalArgumentException();
+        	
+        	if(answer.indexOf(letter) < 0) {wrongGuesHandler(letter); return;}
+        	correctGuesHandler(letter);
+        }
+        catch(IllegalArgumentException ilex)
+        {
+        	errorPopupHandler(inputLenght, "Voer maximaal 1 letter in", "Voer mininaal 1 letter in");
+        }
+    }
+
+    public void checkWord()
+    {
+    	if(gameRunningStatus == false) {errorPopupHandler(0, "", "De game is niet meer actief. \nVoer een nieuw woord in om de game te starten."); return;}
+    	if(txtTryAWord.getText().matches(".*\\d.*")) {errorPopupHandler(0, "", "Het woord mag alleen letters bevatten"); return;}
+    	if(answer.length() <= 0) {errorPopupHandler(0, "", "Voer een woord in om te raden"); return;}
+
+    	String word = txtTryAWord.getText();
+        int inputLenght = word.length();
+        
+        try 
+        {
+        	if(inputLenght < 3) throw new IllegalArgumentException();
+        	
+        	if(!word.equalsIgnoreCase(answer)) {loseGameHandler(""); return;}
+        	winGameHandler();
+        }
+        catch(IllegalArgumentException ilex)
+        {
+        	errorPopupHandler(inputLenght, "Voer minimaal 1 woord in van 3 letters", "Voer minimaal 1 woord in van 3 letters");
+        }
+    }
+    
+    //Helper functions
+    private void correctGuesHandler(String letter)
+    {
+    	StringBuilder fillCorrectLetter = new StringBuilder(correctLetters);
+    	char correctCharLetter = letter.charAt(0);
+    	
+    	long amountCharInString = answer.length() - (correctLetters.chars().filter(ch -> ch != correctCharLetter).count());
+    	int correctLetterIndex = answer.indexOf(letter);
+
+    	if(correctLetters.indexOf(letter) >= 0)
+    	{
+    		
+    		for(int i = 0; i < amountCharInString + 1; i++)
+        	{
+        		if(i == 0)
+        		{
+        			correctLetterIndex = answer.indexOf(letter);
+        		}
+        		else if(i > 0)
+        		{
+        			correctLetterIndex = answer.indexOf(letter, answer.indexOf(letter) + i);
+        		}
+        	}
+    	}
+
+    	if(correctLetterIndex < 0) {wrongGuesHandler(letter); return;}
+    	fillCorrectLetter.setCharAt(correctLetterIndex, correctCharLetter);
+    	correctLetters = fillCorrectLetter.toString();
+    	
+    	lblLettersCorrect.setText("Letters correct: " + correctLetters);
+		txtTryALetter.clear();
+    }
+    
+    private void wrongGuesHandler(String letter)
+    {
+		mistakes++;
+		if(mistakes >= 11) {loseGameHandler("lastImg"); return;}
```

Comment:
```suggestion
		if(mistakes >= maxMistakes) 
		{
	  	    HandleLose("lastImg"); 
		    return;
		}
```

is HandleLose niet een betere naam want het is geen handler maar een function call dus je handled al iets. 
en ik raad je aan om variables voor alles aan te maken ook je maxMistakes dit zorgt dat je later makkelijk meer kansen kan toevoegne. let op dit kan wel voor errors zorgen als je 12 zegt maar je hebt geen 12 images.